ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -1,7 +1,33 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.8
```

Comment:
This is what's currently in the root OLM/registry Dockerfile(s) for the tag being specified. We'll likely need to update this to use the more explicit rhel-8-golang-1.15-openshift-4.8 at some point.

Reply:
Ah, nevermind, there's a mismatch between what base image tags we specify in the registry vs. OLM Dockerfile(s). Should be fine now.